lib/content/message/predictions.ex

```diff
@@ -260,6 +266,27 @@ defmodule Content.Message.Predictions do
     end
   end
 
+  @spec new_cars?(Locations.Location.t() | nil) :: boolean()
+  defp new_cars?(nil) do
+    false
+  end
+
+  defp new_cars?(%Locations.Location{
+         multi_carriage_details: multi_carriage_details,
+         route_id: route_id
+       }) do
+    Enum.any?(multi_carriage_details, fn carriage ->
+      # See http://roster.transithistory.org/ for numbers of new cars
+      case Integer.parse(carriage.label) do
+        :error ->
+          false
+
+        {n, _remaining} ->
+          route_id == "Red" and 1900 <= n and n <= 2151
```

Comment:
Are we not replicating the orange line case here?

Reply:
We actually stopped making the new orange line cars announcements a while back but in that change, I just removed the reference to that message ID that was being used in the `Approaching` struct rather than removing the logic in the `consist_is_new?` function. #526 for reference